pkg/archive/archive_test.go

```diff
@@ -83,41 +83,51 @@ func TestUntar(t *testing.T) {
 
 }
```

Comment:
is TestUntar still a useful test? It seems like it's testing the same things TestUnarchive is.